crates/burn-jit/src/kernel/conv/conv2d/base.rs

```diff
@@ -24,6 +21,9 @@ pub enum Conv2dStrategy {
     /// Implicit GEMM implementation of convolution. Lower memory usage but requires CMMA and
     /// has constraints on tensor shape.
     ImplicitGemm,
+    /// Implicit GEMM implementation of convolution. Lower memory usage but requires CMMA and
+    /// has constraints on tensor shape.
+    ImplicitGemmNew,
```

Comment:
Instead of `New` postfix, it could be named `ImplicitGemmLarge` ?

Reply:
I think I want to call it complex instead, since it's not really about size it's about the relative shape and factors like k size, and I also want to experiment with other variations like k splitting, producer consumer, etc. So I think complex is a better descriptor.